Dockerfile

```diff
@@ -56,7 +56,11 @@ RUN echo "#!/usr/bin/with-contenv bash" > /etc/cont-init.d/bootstrap_container &
     echo "sed -i '/^R_LIBS_USER=/c\\R_LIBS_USER=/home/'\${USER}'/.rpckg' /usr/local/lib/R/etc/Renviron" >> /etc/cont-init.d/bootstrap_container && \
     echo "sed -i 's#HOST=#HOST='\${EMR_HOST_NAME}'#g' /etc/odbc.ini" >> /etc/cont-init.d/bootstrap_container && \
     echo "sed -i 's#REPLACEME#'\${LIVY_URL}'#g' /etc/skel/.spark_config.yml" >> /etc/cont-init.d/bootstrap_container && \
-    chmod +x /etc/cont-init.d/bootstrap_container && \
+    echo "#!/usr/bin/bash" > /etc/profile.d/rstudio.sh && \
+    echo "echo \"export HADOOP_HOME=\"/opt/dataworks/hadoop-3.3.1\"\" >> /etc/profile.d/rstudio.sh" >> /etc/cont-init.d/bootstrap_container && \
+    echo "echo \"export JAVA_HOME=\"/usr/lib/jvm/java-11-openjdk-amd64\"\" >> /etc/profile.d/rstudio.sh" >> /etc/cont-init.d/bootstrap_container && \
+    echo "alias run_beeline=\"/opt/dataworks/apache-hive-2.3.9-bin/bin/beeline -u jdbc:hive2://${EMR_HOST_NAME}:10000;AuthMech=6;delegationToken=${JWT_TOKEN}\"" >> /etc/profile.d/rstudio.sh && \
+    chmod +x /etc/cont-init.d/bootstrap_container && chmod +x /etc/profile.d/rstudio.sh && \
```

Comment:
So this wont work here to chmod /etc/profile.d/rstudio.sh since it doesn't yet exists (its only created when bootstrap_container runs at startup. (if you do the above suggestion it should be available when this line executes).

Reply:
Ok. I see you have created it by this stage with just a comment inside it. So the code should still work